bosh/manifest.yml

```diff
@@ -81,3 +79,33 @@ instance_groups:
   instances: 2
   networks:
   - name: default
+
+variables:
+- name: sealer_password
+  type: password
+- name: bosh_ca
+  type: certificate
+  options:
+    is_ca: true
+    common_name: master-bosh
+- name: idp_cert
+  type: certificate
+  options:
+    ca: bosh_ca
+    common_name: idp.fr.cloud.gov
+# This was originally created with:
+# certstrap request-cert --cn idp-encryption-key --passphrase ''
+# They seem to not have the standard BEGIN/END lines on them, though?
```

Comment:
What does this mean? Can we not generate these with bosh interpolate?

Reply:
Well, I don't know what it means.  There are certs in the secrets files that do not have the standard x509 format (they lack the ----BEGIN/END CERTIFICATE/RSA PRIVATE KEY---- headers/footers).  I don't know if this is required or not.  So technically, we cannot create them automatically.  However, that's why I left the warning.  They could be created in a broken way, and then just have those headers/footers removed in the secrets file by hand and technically ought to work.

I was hoping to avoid having to dig into the whole deployment and see WTF was going on with it by putting this warning in.  If you believe that it would be profitable to dig in and make sure it works, I would be happy to do so, but I figured that this would definitely be significant scope creep and thus something to avoid.

Reply:
@LinuxBozo: do you know what's going on here?

Reply:
@jmcarp @timothy-spencer Yes, it has to be stripped of the typical BEGIN/END comments because it gets shoved into the XML that is used for SAML metadata exchange. It has to be the raw cert data. We could possibly do something in the erb in the bosh release to handle this? https://github.com/18F/shibboleth-boshrelease/blob/master/jobs/idp/templates/metadata/idp-metadata.xml.erb#L20